### ISSUE TITLE
Missing Feature: Activate Virtual Keyboard

### ISSUE THREAD

**Issue by Tabaluga** [procount/pinn#578]
Dear @procount ,

please add an option to view virtual keyboard at any time, especially in the browser.

best regards

**Comment by procount** [comment]
The virtual keyboard only works within PINN, I'm afraid. It does not work with external programs like the browser.

**Comment by Tabaluga** [comment]
Thanks. My idea was to control PINN only with the tv remote, but without a virutal keyboard it was indeed not possible to browse some websites, since I can't enter the address. Best regards

**Comment by procount** [comment]
Sorry for that limitation. It would need an external virtual keyboard like Matchbox or something but I'm not sure how to plumb that into PINN's minimal system.
